packages/xstate-react/test/types.test.tsx

```diff
@@ -119,3 +120,24 @@ describe('useMachine', () => {
     noop(App);
   });
 });
+
+it('useMachine types work for machines with a specified id and state with an after property #5008', () => {
+  // https://github.com/statelyai/xstate/issues/5008
+  const cheatCodeMachine = setup({}).createMachine({
+    id: 'cheatCodeMachine',
+    initial: 'disabled',
+    states: {
+      disabled: {
+        after: {}
```

Comment:
This is extremely weird: if I remove `after: {}`, then (without any changes to XState) this type checks without errors 😵